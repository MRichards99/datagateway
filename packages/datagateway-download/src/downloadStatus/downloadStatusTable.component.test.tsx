// import React from 'react';

// import axios from 'axios';

// describe('DownloadStatusTable', () => {});
