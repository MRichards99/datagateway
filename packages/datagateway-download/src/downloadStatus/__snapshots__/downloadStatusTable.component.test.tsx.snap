// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Download Status Table renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
>
  <WithStyles(ForwardRef(Grid))
    item={true}
  >
    <WithStyles(ForwardRef(Paper))
      style={
        Object {
          "height": "calc(100vh - 110px)",
        }
      }
    >
      <WithStyles(VirtualizedTable)
        actions={
          Array [
            [Function],
            [Function],
          ]
        }
        actionsWidth={100}
        columns={
          Array [
            Object {
              "dataKey": "fileName",
              "filterComponent": [Function],
              "label": "downloadStatus.filename",
            },
            Object {
              "dataKey": "transport",
              "filterComponent": [Function],
              "label": "downloadStatus.transport",
            },
            Object {
              "cellContentRenderer": [Function],
              "dataKey": "status",
              "filterComponent": [Function],
              "label": "downloadStatus.status",
            },
            Object {
              "cellContentRenderer": [Function],
              "dataKey": "createdAt",
              "filterComponent": [Function],
              "label": "downloadStatus.createdAt",
            },
          ]
        }
        data={Array []}
        loading={true}
        onSort={[Function]}
        sort={Object {}}
      />
    </WithStyles(ForwardRef(Paper))>
  </WithStyles(ForwardRef(Grid))>
</div>
`;
